sklearn/multioutput.py

```diff
@@ -590,8 +604,14 @@ def fit(self, X, Y, **fit_params):
         del Y_pred_chain
 
         for chain_idx, estimator in enumerate(self.estimators_):
+            message = self._log_message(
+                estimator_idx=chain_idx + 1,
+                n_estimators=len(self.estimators_),
+                processing_msg=f"Adding feature {self.order_[chain_idx]}",
```

Comment:
This is `chain_idx` is related to `Y.shape[1]` (the number of outputs). I think saying "feature" here is a bit confusing. Maybe:

```suggestion
                processing_msg=f"Processing order {self.order_[chain_idx]}",
```

Reply:
This would be similar to Pipeline's verbose mode:

https://github.com/scikit-learn/scikit-learn/blob/2ae08aa6c1dfb12973d856ce6714140e17dbed78/sklearn/pipeline.py#L1167

Reply:
Sorry missed this one!